### ISSUE TITLE
Function configContains in integration_test.go does not work properly

### ISSUE THREAD

**Issue by agajdosi** [minishift/minishift#1133]
Function was meant to cover 4 cases:
1. key has exact value
2. key has not specified value
4. key does not exist
3. key exists with any value

Case number 4 is broken:

```gherkin
  Scenario: Setup-cdk sets RHEL7 iso to be used                                                 # features/test.feature:4
    When executing "minishift config set disk-size 33" succeeds                                 # integration_test.go:454 -> github.com/minishift/minishift/test/integration.executingMinishiftCommandSucceedsOrF
ails
    Then JSON config file "config/config.json" contains key "disk-size" with value "33"         # integration_test.go:309 -> github.com/minishift/minishift/test/integration.configContains
    Then JSON config file "config/config.json" does not contain key "disk-size" with value "66" # integration_test.go:309 -> github.com/minishift/minishift/test/integration.configContains
    Then JSON config file "config/config.json" does not contain key "nonsense"                  # integration_test.go:309 -> github.com/minishift/minishift/test/integration.configContains
    Then JSON config file "config/config.json" contains key "disk-size"                         # integration_test.go:309 -> github.com/minishift/minishift/test/integration.configContains
    For key 'disk-size' config contains unexpected value '33'
```

I would like to use it in downstream for `iso-url` value check as part of CDK-142, maybe would be nice to implement regex also.

**Comment by hferentschik** [comment]
> I would like to use it in downstream for iso-url value check as part of CDK-142, maybe would be nice to implement regex also.

+1

**Comment by hferentschik** [comment]
Merged via pull request #1143
